groceries-ng/2.md

```diff
@@ -308,7 +308,9 @@ constructor(private userService: UserService) {
 }
 ```
 
-* **e.** Finally, find the existing `submit()` function in your `AppComponent`, and replace it with the following three functions.
+* **e.** After that, **remove** the existing `submit()` function in your `AppComponent` class. (You’ll add a new one in the next step.)
+
+* **f.** Finally, **add** the the following three functions to your `AppComponent` class.
```

Comment:
three functions -> do you mean "three rows"?

Reply:
No I do mean three functions (you might need to expand in the GitHub interface to see the whole context). Actually though “methods” is probably the better word here. Would that help clarify?

Reply:
I went ahead and switched from “functions” --> “methods”. Let me know if you have any concerns.